mobile/src/people/company-departments-level-animated-node.tsx

```diff
@@ -137,6 +156,13 @@ interface CompanyDepartmentsLevelAnimatedNodeProps {
 
 export class CompanyDepartmentsLevelAnimatedNode extends Component<CompanyDepartmentsLevelAnimatedNodeProps> {
     private readonly animatedContainerOpacity = new Animated.Value(0);
+    private readonly isAndroid70: boolean = false;
+
+    constructor(props: CompanyDepartmentsLevelAnimatedNodeProps, context: any) {
+        super(props, context);
+
+        this.isAndroid70 = Platform.OS === 'android' && Platform.Version === 24;
```

Comment:
Maybe it's better to check for 7.1 (API level 25) too?

Reply:
Done. 1cbf4fa590c9346e2905f848cb264cca0be029d7